### ISSUE TITLE
Terminology

### ISSUE THREAD

**Issue by ghost** [QingyuZhao/BR-Net#4]
Would you mind clearing up some confusion I have about the terminology in the paper and the github example code? This is especially in regards to the difference between the regressor and the distiller which both are referred to the bias predictor at various stages. Is this mapping correct? 

Encoder: Training Data -> Features
Regressor: Features -> Confounders
Distiller: Training data -> Confounders
Classifier: Training Data -> Training Labels

I also do not understand the "Train distiller" section of the train_cf() function as no actual training seems to take place before "Train regressor". 

I would appreciate any clearing up of my confusion. Thank you!

**Comment by QingyuZhao** [comment]
Yes, your interpretation is correct, and I also moved around the comments to clarify
________________________________
